src/components/timeline/index.css

```diff
@@ -28,24 +28,59 @@
   list-style: none;
 }
 
-.timelineToggle {
-  position: absolute;
-  top: 0;
-  left: -150px;
+.timelineSettings {
+  display: flex;
+  height: 25px;
+  flex-flow: row nowrap;
+  padding: 0;
+  border-bottom: 1px solid var(--grey-30);
+  color: var(--ink-70);
+  line-height: 25px;
+  white-space: nowrap;
+}
+
+.timelineSettingsToggle {
   display: flex;
-  width: 150px;
-  height: 22px;
-  font-size: 9px;
+  padding: 0 8px;
+  border-right: 1px solid var(--grey-30);
 }
 
-.timelineToggleInput {
-  width: 12px;
-  height: 12px;
-  margin: 0 3px;
+.timelineSettingsToggleInput {
+  margin: 0 5px;
   vertical-align: middle;
 }
 
-.timelineToggleLabel {
+.timelineSettingsToggleLabel {
   display: flex;
   align-items: center;
 }
+
+.timelineSettingsHiddenTracks {
+  position: relative;
+  height: 17px;
+  padding: 0 7px 0 20px;
+  margin: 4px 5px 0;
+  border: 0;
+  background-color: var(--blue-60);
+  border-radius: 12px;
+  color: #fff;
+  line-height: 17px;
```

Comment:
please add `font: inherit` to override the default button font and get the one from the parent, otherwise the font is way too big, at least for me.

The alternative is to make this button a photon-button and inherit all the nice things that are already defined. You'd have to implement also a photon-button-primary to set the colors for all states (including hover and pressed) for this button type.

Reply:
I'd like to keep the style implementation separate, as I'm scared of overriding an existing ruleset with custom properties.